starfish/src/main/java/org/maxandersen/Starfish.java

```diff
@@ -0,0 +1,238 @@
+            else
+            if(id==2){ide="eclipse";System.out.println("Selected IDE:Eclipse");break;}
+            else
+            if(id==3){ide="idea";System.out.println("Selected IDE:IntelliJ_IDEA");break;}
+            else
+            System.out.println("--------Invalid Input Try Again--------");
+            }
+
+            //-----------Now we'll get preferred clone path on local file system from user--------------
+            System.out.println("--------Enter preferred Clonning path--------");
+            clone_path=reader.readLine();
+
+            //----------Now we'll write configurations to the YAML FILE------------
+            ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+            Config configuration=new Config(ide,clone_path);
+            mapper.writeValue(configFile, configuration); //Writing to YAML File
+            
+        
+        
+    }
+    catch (Exception e) {
+        e.printStackTrace();
+        System.out.print("Press any key to continue . . . ");
+        reader.readLine();
+    }
+
+}
+
+public static void gitClone(Path directory, String originUrl) throws IOException, InterruptedException {
+    //Function for git clonning
+    runCommand(directory.getParent(), "git", "clone", originUrl, directory.getFileName().toString());
+}
+
+
+
+
+public static void runCommand(Path directory, String... command) throws IOException, InterruptedException {
+    //Function to Run Commands using Process Builder
+    
+    Objects.requireNonNull(directory, "directory");
+    if (!Files.exists(directory)) {
+        throw new RuntimeException("can't run command in non-existing directory '" + directory + "'");
+    }
+    ProcessBuilder pb = new ProcessBuilder()
+            .command(command)
+            .directory(directory.toFile());
+    Process p = pb.start();
+    StreamGobbler errorGobbler = new StreamGobbler(p.getErrorStream(), "E");
+    StreamGobbler outputGobbler = new StreamGobbler(p.getInputStream(), "O");
+    outputGobbler.start();
+    errorGobbler.start();
+    int exit = p.waitFor();
+    errorGobbler.join();
+    outputGobbler.join();
+    if (exit != 0) {
+        throw new AssertionError(String.format("runCommand returned %d", exit));
+    }
+}
+
+
+
+private static class StreamGobbler extends Thread {
+
+    private final InputStream is;
+    private final String type;
+
+    private StreamGobbler(InputStream is, String type) {
+        this.is = is;
+        this.type = type;
+    }
+
+    @Override
+    public void run() {
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(is));) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                System.out.println(type + "> " + line);
+            }
+        } catch (IOException ioe) {
+            ioe.printStackTrace();
+            
+        }
+    }
+}
+
+
+//Function to fetch all installed packages on System
+public static void fetch_installed_packages()throws Exception{
```

Comment:
This method are not used any where and are Linux distribution specific. Let's not put such code in before I actually has a use.

Reply:
Will remove that

Reply:
done